docs/cloud-native-security/cloud-nat-sec-kubernetes-dashboard.asciidoc

```diff
@@ -53,6 +53,8 @@ To collect session data for the dashboard, you'll deploy a Kubernetes DaemonSet
 |**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | Container-optimized OS (COS), Ubuntu
 |=====================
 
+NOTE: When running within Kubernetes, {elastic-endpoint}'s <<host-isolation-ov,Host Isolation>> and <<malware-proection,Malware Protection>> features are disabled.
```

Comment:
I'm fairly certain that neither features need to be capitalized when they're generally mentioned in the docs. @joepeeples thoughts?

Reply:
```suggestion
NOTE: When running within Kubernetes, {elastic-endpoint}'s <<host-isolation-ov,host isolation>> and <<malware-proection,malware protection>> features are disabled.
```

Reply:
+1 to lowercase, which has already been implemented 👍